extension/src/lib/highContrast.js

```diff
@@ -0,0 +1,87 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+// This component makes use of css/highContrast.css to perform the adaptations
+// of the web content, and this is done through chrome.tabs.executeScript.
+//
+fluid.defaults("gpii.chrome.highContrast", {
+    gradeNames: ["fluid.modelComponent"],
+    scriptTemplate: "document.documentElement.setAttribute('data-gpii-hc','__THEME__');" +
```

Comment:
Yes, please place these in files. As well as this, please return to the use of function calls with arguments (an accepted means of parameterising functionality since 1951) rather than template string substitution within code.